src/main/resources/db/migration/0200/V227__PlantingSeasons.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tracking.planting_seasons (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    planting_site_id BIGINT NOT NULL REFERENCES tracking.planting_sites ON DELETE CASCADE,
+    start_date DATE NOT NULL,
+    start_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    end_date DATE NOT NULL,
+    end_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    is_active BOOLEAN NOT NULL,
+
+    CHECK (start_date < end_date),
```

Comment:
Should we implement the planting season restrictions here? Or are those validations elsewhere? For example "planting season must be at least 4 weeks" and "planting season is no longer than 1 year".